pytorch_lightning/strategies/parallel.py

```diff
@@ -42,6 +43,7 @@ def __init__(
         super().__init__(accelerator=accelerator, checkpoint_io=checkpoint_io, precision_plugin=precision_plugin)
         self.parallel_devices = parallel_devices
         self.cluster_environment = cluster_environment
+        self._layer_sync: Optional[LayerSync] = None
```

Comment:
Just a question: Why is this plugin private if other plugins (like cluster env and precision) are not?

Reply:
I think mainly as a security because we set it from the accelerator connector. We may also want to expose it as a constructor arg in the future because the others are too. Just trying to give us max flexibility after 1.6 when we declare it as stable. We can always add new optional args and properties :)